jmx-metrics/src/main/resources/target-systems/hadoop.groovy

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+def beanHadoopNameNodeFS = otel.mbean("Hadoop:service=NameNode,name=FSNamesystem")
+otel.instrument(beanHadoopNameNodeFS, "hadoop.hdfs.disk.usage", "The amount of disk used by data nodes.", "by",
```

Comment:
Can we track with the descriptions used in the docs? They use `Current used capacity across all DataNodes in bytes` so that "disk" doesn't seem directly applicable. It may also be preferable to identify these metrics as originating from a name node so that the `node_name` attribute is clearer.

Reply:
I agree. The origin probably makes more sense than the category for the metric namespace. I've updated the metrics to replace `hdfs` with `name_node`. And I've also replaced `disk` with the `capacity` terminology.